bindings/jupyter/docs/manifold.md

```diff
@@ -53,15 +54,15 @@ data = {
   x:     [...],         # feature data
   yPred: [[...], ...]   # prediction data
   yTrue: [...],         # ground truth data
-}
+};
```

Comment:
The code block is python, so you likely want no semi-colons

Reply:
addressed.